README.md

```diff
@@ -88,8 +87,10 @@ Chapter 3 - Vectors
 Chapter 4 - Subsetting
 
 - Cohort 1: [Scott Nestler](https://twitter.com/ScottNestler) ([slides](https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week04/Cohort1/Chap4slides.html#1) | [video](https://www.youtube.com/watch?v=eLMpCc0t1cg))
+- Cohort 2: ...
+- Cohort 3: ... [video](https://www.youtube.com/watch?v=CvvXkXiF3Ig)
```

Comment:
Just reviewing Shel's slides here, does it make sense to credit her and Alan? (I wasn't around to watch the presentation)

Reply:
Can also include links to slide (https://r4ds.github.io/bookclub-Advanced_R/Presentations/Week04/Cohort3/chapter4.html#1)

Reply:
Yes. They worked together on the slides and presented together.

Reply:
Thanks. I wanted to make another pull request to add the slides link to the readme, but I guess Tan you can do that for me? Or should I do it?

Reply:
I just got Novica to do it 😎 good job today!